### ISSUE TITLE
XCode 14.3: rsync error: some files could not be transferred

### ISSUE THREAD

**Issue by cjohn001** [NativeScript/NativeScript#10259]
### Issue Description

Hello together,
with the ns 8.5 release I cannot build and run my app on a real ios device anymore. Deploying and debugging on the emulator works. However, when building for a real device I see the error messages below. The Frameworks are indeed not available at the given path. Anyone an idea how to work around the issue?

I deleted the derived data folders and rebuild it. This did not help.

```
5 warnings generated.
mkdir -p /Users/cjohn/Entwicklung/MND/my-nutri-diary/mnd/platforms/ios/build/Debug-iphoneos/mnd.app/Frameworks
Symlinked...
rsync --delete -av --filter P .*.?????? --links --filter "- CVS/" --filter "- .svn/" --filter "- .git/" --filter "- .hg/" --filter "- Headers" --filter "- PrivateHeaders" --filter "- Modules" "../../../../../../../../../Library/Developer/Xcode/DerivedData/mnd-gqojzlqggpihlfgewdbzuefoegom/Build/Intermediates.noindex/ArchiveIntermediates/mnd/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/GTMSessionFetcher.framework" "/Users/cjohn/Library/Developer/Xcode/DerivedData/mnd-gqojzlqggpihlfgewdbzuefoegom/Build/Intermediates.noindex/ArchiveIntermediates/mnd/InstallationBuildProductsLocation/Applications/mnd.app/Frameworks"
building file list ... rsync: link_stat "/Users/cjohn/Entwicklung/MND/my-nutri-diary/mnd/platforms/ios/../../../../../../../../../Library/Developer/Xcode/DerivedData/mnd-gqojzlqggpihlfgewdbzuefoegom/Build/Intermediates.noindex/ArchiveIntermediates/mnd/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/GTMSessionFetcher.framework" failed: No such file or directory (2)
done

sent 29 bytes  received 20 bytes  98.00 bytes/sec
total size is 0  speedup is 0.00
rsync error: some files could not be transferred (code 23) at /AppleInternal/Library/BuildRoots/9e200cfa-7d96-11ed-886f-a23c4f261b56/Library/Caches/com.apple.xbs/Sources/rsync/rsync/main.c(996) [sender=2.6.9]
Command PhaseScriptExecution failed with a nonzero exit code
note: Building targets in dependency order
note: Run script build phase 'NativeScript PreLink' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'mnd' from project 'mnd')
note: Run script build phase 'NativeScript PreBuild' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'mnd' from project 'mnd')
note: Run script build phase 'NativeScript PostBuild' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'mnd' from project 'mnd')
** ARCHIVE FAILED **

Unable to apply changes on device: 
```

### Environment

```yaml
OS: macOS 13.2.1
CPU: (10) arm64 Apple M1 Pro
Shell: /bin/zsh
node: 18.12.1
npm: 8.19.2
nativescript: 8.5.1

# android
java: 11.0.18
ndk: Not Found
apis: Not Found
build_tools: Not Found
system_images: Not Found

# ios
xcode: 14.3/14E222b
cocoapods: 1.12.0
python: 3.11.2
python3: 3.11.2
ruby: 2.7.7
platforms: 
  - DriverKit 22.4
  - iOS 16.4
  - macOS 13.3
  - tvOS 16.4
  - watchOS 9.4
```

### Dependencies

```json
"dependencies": {
  "@angular/animations": "15.2.4",
  "@angular/common": "15.2.4",
  "@angular/compiler": "15.2.4",
  "@angular/core": "15.2.4",
  "@angular/forms": "15.2.4",
  "@angular/platform-browser": "15.2.4",
  "@angular/platform-browser-dynamic": "15.2.4",
  "@angular/router": "15.2.4",
  "@apollo/client": "3.7.10",
  "@mnd/external-web-view": "file:../mnd-plugins/dist/packages/external-web-view/mnd-external-web-view-1.0.0.tgz",
  "@nativescript/angular": "15.2.0",
  "@nativescript/core": "8.5.0",
  "@nativescript/iqkeyboardmanager": "2.1.1",
  "@nativescript/localize": "5.0.4",
  "@nativescript/mlkit-barcode-scanning": "2.0.0",
  "@nativescript/mlkit-core": "2.0.0",
  "@nativescript/secure-storage": "3.0.0",
  "@nativescript/theme": "3.0.2",
  "@nativescript/ui-charts": "0.2.4",
  "apollo-angular": "4.2.1",
  "apollo3-cache-persist": "0.14.1",
  "d3-ease": "3.0.1",
  "graphql": "16.6.0",
  "graphql-tag": "2.12.6",
  "intl": "1.2.5",
  "moment": "2.29.4",
  "nativescript-health-data": "file:../mnd-custom-plugins/nativescript-health-data/publish/package/nativescript-health-data-2.0.0.tgz",
  "nativescript-oauth2-ext": "file:../mnd-custom-plugins/nativescript-oauth2-ext/publish/package/nativescript-oauth2-ext-3.0.1.tgz",
  "nativescript-sqlite": "2.8.6",
  "nativescript-ui-calendar": "15.2.3",
  "nativescript-ui-gauge": "15.2.3",
  "rxjs": "7.8.0",
  "uuid": "9.0.0",
  "zone.js": "0.13.0"
},
"devDependencies": {
  "@angular-devkit/build-angular": "15.2.4",
  "@angular/compiler-cli": "15.2.4",
  "@graphql-codegen/cli": "3.2.2",
  "@graphql-codegen/fragment-matcher": "4.0.1",
  "@graphql-codegen/introspection": "3.0.1",
  "@graphql-codegen/typescript": "3.0.2",
  "@graphql-codegen/typescript-apollo-angular": "3.5.6",
  "@graphql-codegen/typescript-operations": "3.0.2",
  "@nativescript/android": "8.5.0",
  "@nativescript/ios": "8.5.1",
  "@nativescript/types": "8.5.0",
  "@nativescript/webpack": "5.0.14",
  "@ngtools/webpack": "15.2.4",
  "@types/d3-ease": "3.0.0",
  "@types/intl": "1.2.0",
  "@types/node": "18.15.11",
  "@types/uuid": "9.0.1",
  "keycloak-request-token": "0.1.0",
  "rimraf": "4.4.1",
  "sass": "1.60.0",
  "ts-node": "10.9.1",
  "typescript": "4.9.5"
}
```

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by rigor789** [comment]
This is a cocoapod related issue with XCode 14.3 see: https://github.com/CocoaPods/CocoaPods/pull/11828/files

Making this change locally as far as I understand fixes it until a proper update is release by Cocoapods.

**Comment by cjohn001** [comment]
@rigor789 : Thanks a lot, the workaround does its job!

**Comment by ray007** [comment]
Any hint on how to work around this problem locally before an upstream fix is available?

**Comment by rigor789** [comment]
@ray007 the linked issue has some context - but sounds like they're close to releasing a cocoapods update... Until then I'm sticking with XCode 14.2 personally. 

(I'm using https://github.com/RobotsAndPencils/XcodesApp and never rely on AppStore for XCode, especially not auto-updates)

**Comment by cjohn001** [comment]
@ray007
I just changed the file in my local installation:

lib/cocoapods/generator/embed_frameworks_script.rb

the single line as you can see from the patch was sufficient to change for me
source="$(readlink -f "${source}")"

**Comment by alexonozor** [comment]
@cjohn001 To be sure we are having the same issue our app is crashing on a real device only on Ios OS 14.x.
It crashed upon startup.

**Comment by HoncoTech** [comment]
Xcode version 14.3 (14E222b) + iOS 16.4 + MacOSX Ventura 13.3 , I am not able to build the nativescript project. The tns-ios version is 6.5.3. Getting below error-

error: Build input file cannot be found: '/Users/xxxxxxxxx/yyyyyy/zzzzzzz/platforms/ios/build/Debug-iphonesimulator/metadata-x86_64.bin'. Did you forget to declare this file as an output of a script phase or custom build rule which produces it? (in target 'XXXXXXXX' from project 'YYYYYYYY')

**Comment by codealvarez** [comment]
You're right @rigor789 - It works if you use Xcode 14.2, thank you - I had to install [Xcodes](https://github.com/RobotsAndPencils/XcodesApp) to change my Xcode version without uninstall Xcode 14.3 - I'm using `"@nativescript/ios": "8.3.3",` and everything goes well now. Thank you!

**Comment by vicmasa** [comment]
Hello.
I have the same problem, version 8.5.2 does not support @nativescript/ui-charts

```yaml
OS: macOS 13.2.1
CPU: (8) arm64 Apple M1 Pro
Shell: /bin/zsh
node: 19.8.1
npm: 9.5.1
nativescript: 8.5.2

# android
java: 11.0.17
ndk: Not Found
apis: Not Found
build_tools: Not Found
system_images: Not Found

# ios
xcode: 14.3/14E222b
cocoapods: 1.12.0
python: 2.7.18
python3: 3.9.6
ruby: 2.7.8
platforms: 
  - DriverKit 22.4
  - iOS 16.4
  - macOS 13.3
  - tvOS 16.4
  - watchOS 9.4
```

### Dependencies

```json
"dependencies": {
  "@nativescript/core": "~8.5.0",
  "@nativescript/theme": "~3.0.2",
  "@nativescript/ui-charts": "^0.2.4"
},
"devDependencies": {
  "@nativescript/ios": "8.5.1",
  "@nativescript/webpack": "~5.0.14",
  "webpack-cli": "^5.0.1"
}
```

```
Searching for devices...
Preparing project...
webpack is watching the files...
asset vendor.js 6.74 MiB [compared for emit] (name: vendor) (id hint: defaultVendor)
asset bundle.js 344 KiB [compared for emit] (name: bundle)
asset runtime.js 86.2 KiB [compared for emit] (name: runtime)
asset tns_modules/inspector_modules.js 1.22 KiB [compared for emit] (name: tns_modules/inspector_modules)
Entrypoint bundle 7.16 MiB = runtime.js 86.2 KiB vendor.js 6.74 MiB bundle.js 344 KiB
Entrypoint tns_modules/inspector_modules 6.82 MiB = runtime.js 86.2 KiB vendor.js 6.74 MiB tns_modules/inspector_modules.js 1.22 KiB
runtime modules 24.8 KiB 11 modules
orphan modules 302 bytes [orphan] 5 modules
modules by path ./node_modules/@nativescript/ 1.61 MiB 195 modules
modules by path ./node_modules/css-tree/ 317 KiB 113 modules
modules by path ./app/ 123 KiB 7 modules
modules by path ./node_modules/source-map/lib/*.js 36.3 KiB 6 modules
modules by path ./node_modules/reduce-css-calc/dist/ 131 KiB 5 modules
modules by path ./node_modules/postcss-value-parser/lib/*.js 12.5 KiB
  ./node_modules/postcss-value-parser/lib/index.js 607 bytes [built] [code generated]
  + 4 modules
modules by path ./node_modules/mdn-data/css/*.json 242 KiB
  ./node_modules/mdn-data/css/at-rules.json 11.6 KiB [built] [code generated]
  + 2 modules
+ 5 modules
webpack 5.78.0 compiled successfully in 1469 ms
Webpack compilation complete. Watching for file changes.
File change detected. Starting incremental webpack compilation...
assets by status 7.16 MiB [cached] 4 assets
Entrypoint bundle 7.16 MiB = runtime.js 86.2 KiB vendor.js 6.74 MiB bundle.js 344 KiB
Entrypoint tns_modules/inspector_modules 6.82 MiB = runtime.js 86.2 KiB vendor.js 6.74 MiB tns_modules/inspector_modules.js 1.22 KiB
cached modules 2.68 MiB (javascript) 24.8 KiB (runtime) [cached] 354 modules
./app/ sync .(xml%7Cjs%7Cs?css)$ 270 bytes [built]
webpack 5.78.0 compiled successfully in 74 ms
Webpack compilation complete. Watching for file changes.
Installing pods...
Analyzing dependencies
Downloading dependencies
Installing Highcharts (9.2.2)
Generating Pods project
Integrating client project

[!] Please close any current Xcode sessions and use `ns852.xcworkspace` for this project from now on.
Pod installation complete! There is 1 dependency from the Podfile and 1 total pod installed.
Updating runtime package.json with configuration values...
Project successfully prepared (ios)
Building project...
Xcode build...
2023-04-06 17:28:31.650 xcodebuild[10080:74445] DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)
./.build_env_vars.sh: line 508: declare: UID: readonly variable
NSLD: Swift bridging header '*-Swift.h' not found under '/Users/victor/Library/Developer/Xcode/DerivedData/ns852-gemgmjvfbbdqyvefnutdiqvbnwyi/Build/Intermediates.noindex/ns852.build/Debug-iphonesimulator/ns852.build/Objects-normal/arm64'
Generating metadata...~/invian/ns/ns852/platforms/ios/internal/metadata-generator-arm64/bin ~/invian/ns/ns852/platforms/ios
Python version: 3.9.6 (default, Mar 10 2023, 20:16:38) 
[Clang 14.0.3 (clang-1403.0.22.14.1)]
Generating metadata for arm64
Metadata Generator Arguments: 
./objc-metadata-generator -verbose -output-bin /Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/metadata-arm64.bin -output-umbrella /Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/umbrella-arm64.h -docset-path /Users/victor/Library/Developer/Shared/Documentation/DocSets/com.apple.adc.documentation.iOS.docset Xclang -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator16.4.sdk -std=gnu99 -target arm64-apple-ios16.4-simulator -I/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/include -I/Users/victor/invian/ns/ns852/platforms/ios/internal/Swift-Modules -I/Users/victor/invian/ns/ns852/platforms/ios/internal -I/Users/victor/invian/ns/ns852/platforms/ios/../../App_Resources/iOS/src -I/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/core/platforms/ios/src -I/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/ui-charts/platforms/ios/src -I/Users/victor/invian/ns/ns852/platforms/ios/../../App_Resources/iOS/src -F/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator -F/Users/victor/invian/ns/ns852/platforms/ios/internal/ -F/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator -F/Users/victor/invian/ns/ns852/platforms/ios/Pods/Highcharts/development -F/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/core/platforms/ios -DCOCOAPODS=1 -DDEBUG=1 
Clang Arguments: 
"-v", "-x", "objective-c", "-fno-objc-arc", "-fmodule-maps", "-ferror-limit=0", "-Wno-unknown-pragmas", "-Wno-ignored-attributes", "-Wno-nullability-completeness", "-Wno-expansion-to-defined", "-D__NATIVESCRIPT_METADATA_GENERATOR=1", "-isysroot", "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator16.4.sdk", "-std=gnu99", "-target", "arm64-apple-ios16.4-simulator", "-I/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/include", "-I/Users/victor/invian/ns/ns852/platforms/ios/internal/Swift-Modules", "-I/Users/victor/invian/ns/ns852/platforms/ios/internal", "-I/Users/victor/invian/ns/ns852/platforms/ios/../../App_Resources/iOS/src", "-I/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/core/platforms/ios/src", "-I/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/ui-charts/platforms/ios/src", "-I/Users/victor/invian/ns/ns852/platforms/ios/../../App_Resources/iOS/src", "-F/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator", "-F/Users/victor/invian/ns/ns852/platforms/ios/internal/", "-F/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator", "-F/Users/victor/invian/ns/ns852/platforms/ios/Pods/Highcharts/development", "-F/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/core/platforms/ios", "-DCOCOAPODS=1", "-DDEBUG=1", 
Merged 1094 categories.
Result: 55841 declarations from 174 top level modules
Done! Running time: 4.42392 sec 
Saving metadata generation's stderr stream to: /Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/metadata-generation-stderr-arm64.txt
Generating metadata for x86_64
Metadata Generator Arguments: 
./objc-metadata-generator -verbose -output-bin /Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/metadata-x86_64.bin -output-umbrella /Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/umbrella-x86_64.h -docset-path /Users/victor/Library/Developer/Shared/Documentation/DocSets/com.apple.adc.documentation.iOS.docset Xclang -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator16.4.sdk -std=gnu99 -target x86_64-apple-ios16.4-simulator -I/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/include -I/Users/victor/invian/ns/ns852/platforms/ios/internal/Swift-Modules -I/Users/victor/invian/ns/ns852/platforms/ios/internal -I/Users/victor/invian/ns/ns852/platforms/ios/../../App_Resources/iOS/src -I/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/core/platforms/ios/src -I/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/ui-charts/platforms/ios/src -I/Users/victor/invian/ns/ns852/platforms/ios/../../App_Resources/iOS/src -F/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator -F/Users/victor/invian/ns/ns852/platforms/ios/internal/ -F/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator -F/Users/victor/invian/ns/ns852/platforms/ios/Pods/Highcharts/development -F/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/core/platforms/ios -DCOCOAPODS=1 -DDEBUG=1 
Clang Arguments: 
"-v", "-x", "objective-c", "-fno-objc-arc", "-fmodule-maps", "-ferror-limit=0", "-Wno-unknown-pragmas", "-Wno-ignored-attributes", "-Wno-nullability-completeness", "-Wno-expansion-to-defined", "-D__NATIVESCRIPT_METADATA_GENERATOR=1", "-isysroot", "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator16.4.sdk", "-std=gnu99", "-target", "x86_64-apple-ios16.4-simulator", "-I/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/include", "-I/Users/victor/invian/ns/ns852/platforms/ios/internal/Swift-Modules", "-I/Users/victor/invian/ns/ns852/platforms/ios/internal", "-I/Users/victor/invian/ns/ns852/platforms/ios/../../App_Resources/iOS/src", "-I/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/core/platforms/ios/src", "-I/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/ui-charts/platforms/ios/src", "-I/Users/victor/invian/ns/ns852/platforms/ios/../../App_Resources/iOS/src", "-F/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator", "-F/Users/victor/invian/ns/ns852/platforms/ios/internal/", "-F/Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator", "-F/Users/victor/invian/ns/ns852/platforms/ios/Pods/Highcharts/development", "-F/Users/victor/invian/ns/ns852/platforms/ios/../../node_modules/@nativescript/core/platforms/ios", "-DCOCOAPODS=1", "-DDEBUG=1", 
Merged 1094 categories.
Result: 55897 declarations from 175 top level modules
Done! Running time: 4.04598 sec 
Saving metadata generation's stderr stream to: /Users/victor/invian/ns/ns852/platforms/ios/build/Debug-iphonesimulator/metadata-generation-stderr-x86_64.txt
~/invian/ns/ns852/platforms/ios
ld: building for iOS Simulator, but linking in dylib built for iOS, file '/Users/victor/invian/ns/ns852/platforms/ios/Pods/Highcharts/development/Highcharts.framework/Highcharts' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
note: Using codesigning identity override: 
note: Building targets in dependency order
note: Removed stale file '/Users/victor/Library/Developer/Xcode/DerivedData/ns852-gemgmjvfbbdqyvefnutdiqvbnwyi/Build/Intermediates.noindex/ns852.build/Debug-iphonesimulator/ns852.build/Script-26D215E5777F7D77E97FAB43.sh'

warning: Stale file '/Users/victor/invian/ns/ns852/platforms/ios/build/sharedpch/SharedPrecompiledHeaders/333190205407824326/ns852-Prefix.pch.gch' is located outside of the allowed root paths.

warning: Stale file '/Users/victor/invian/ns/ns852/platforms/ios/build/sharedpch/SharedPrecompiledHeaders/5045094958533541288/ns852-Prefix.pch.gch' is located outside of the allowed root paths.

note: Run script build phase 'NativeScript PreBuild' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'ns852' from project 'ns852')
note: Run script build phase 'NativeScript PreLink' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'ns852' from project 'ns852')
note: Run script build phase 'NativeScript PostBuild' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked. (in target 'ns852' from project 'ns852')
** BUILD FAILED **

Unable to apply changes on device: 3A4FF99C-6D5A-4AB9-931E-9C3B6AB917CB. Error is: Command xcodebuild failed with exit code 65.
```

**Comment by NathanWalker** [comment]
For anyone in this situation you can run these commands:
```
gem install specific_install
gem specific_install -l https://github.com/CocoaPods/CocoaPods.git
```
to clear things up.

**Comment by NathanWalker** [comment]
Hi @vicmasa sure thing. Here's what you can do to solve your error:

1. Ensure package is updated to reflect latest minor: `"@nativescript/google-maps": "^1.5.0",` ... this 1.5.0 was just published this morning to reflect a pod update from googlemaps compat with arm64.

2. Update `App_Resources/iOS/Podfile` with the following:
```
platform :ios, '13.0'

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '13.0'
      config.build_settings['CODE_SIGNING_ALLOWED'] = 'NO'
    end
  end
end
```

3. Update `App_Resources/iOS/build.xcconfig` to at least include a matching version target line:
```
IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

4. `ns clean` and now run.

**Comment by rigor789** [comment]
The issue has been resolved in [CocoaPods 1.12.1](https://github.com/CocoaPods/CocoaPods/releases/tag/1.12.1).

Update your CocoaPods:
```bash
gem install cocoapods # depending on your system, might need to run with sudo
```

Then `ns clean` and re-run your projects.
